include/SDL3/SDL_audio.h

```diff
@@ -1008,12 +1008,12 @@ typedef struct _SDL_AudioStream SDL_AudioStream;
  * \sa SDL_AudioStreamClear
  * \sa SDL_FreeAudioStream
  */
-extern DECLSPEC SDL_AudioStream * SDLCALL SDL_NewAudioStream(const SDL_AudioFormat src_format,
```

Comment:
making these arguments const changes the API!

(and breaks the build with MSVC, because now this signature doesn't match the one of the implementation and  the dynapi stuff)